english/watch-star-wars/WWIII.md

```diff
@@ -0,0 +1,3 @@
+After the wall is completed, President Trump sends Mexico a bill for $8 billion. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.